python/init.sls

```diff
@@ -49,7 +49,8 @@ setuptools:
 
 pip:
   pip.installed:
-    - upgrade: True
+    - name: pip==8.1.1
+    - upgrade: False
```

Comment:
Not sure the upgrade: False is needed? For the specialtyfood project just replacing updgrade True with the pinned pip line in the local override of this file worked...